Example/PaymentSheet Example/PaymentSheet Example/Resources/Assets.xcassets/AppIcon.solidimagestack/Front.solidimagestacklayer/Content.imageset/Contents.json

```diff
@@ -0,0 +1,13 @@
+{
+  "images" : [
+    {
```

Comment:
Are these supposed to be in here?

Reply:
Oof good catch, committed these by mistake